diff --git a/usr/local/lib/node_modules/@semantic-release/github/lib/publish.js b/usr/local/lib/node_modules/@semantic-release/github/lib/publish.js
index df0fa48..0ff9234 100644
--- a/usr/local/lib/node_modules/@semantic-release/github/lib/publish.js
+++ b/usr/local/lib/node_modules/@semantic-release/github/lib/publish.js
@@ -15,7 +15,7 @@ module.exports = async (pluginConfig, context) => {
     nextRelease: {gitTag, notes},
     logger,
   } = context;
-  const {githubToken, githubUrl, githubApiPathPrefix, proxy, assets} = resolveConfig(pluginConfig, context);
+  const {githubToken, githubUrl, githubApiPathPrefix, proxy, assets, draft} = resolveConfig(pluginConfig, context);
   const {name: repo, owner} = parseGithubUrl(repositoryUrl);
   const github = getClient({githubToken, githubUrl, githubApiPathPrefix, proxy});

@@ -27,6 +27,7 @@ module.exports = async (pluginConfig, context) => {
     name: gitTag,
     target_commitish: branch,
     body: notes,
+    draft: draft,
   };
   /* eslint-enable */


